portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
```

Comment:
Given the purpose of this function, I would've renamed it to something like getRandomGreeting/fact.